game/script-jokes.rpy

```diff
@@ -631,3 +659,74 @@ label joke_con_crete:
     extend 1nchgn "-crete!"
 
     return
+
+label joke_footless_snakes:
+    n 1ulraj "Hey,{w=0.2} [player]...{w=1}{nw}"
+    extend 1tnmfl " have you ever wondered how to measure snakes?"
+    n 1uwdaj "Like they are so long!"
+    n 1fsqcs "What measurement could be possibly the best?"
+    n 1fcsss "Well,{w=0.2} my guess would be in inches or centimetre...{w=0.5}"
+    n 1nsrss "Because they...{w=1}{nw}"
+    extend 1nchgn " have no {i}feet{/i}!"
```

Comment:
```suggestion
    n 1fcsss "Well,{w=0.2} I guess you'd just be stuck with metric...{w=0.5}"
    n 1fsqbg "...'Cause you definitely aren't using {w=0.5}{nw}"
    extend 1nchgn " {i}feet{/i}!"
```

I think picking up on the metric/imperial difference here really helps with the punchline rather than calling out individual units: we might even tease a few users with it as a bonus! I stuck with using feet as the punchline, tying it into the metric mention because it worked really well!